### PR TITLE
add complete command as recap

### PR DESCRIPTION
**Pull request by ninewise**
@bmesuere would you happen to know the RAM type in Adler?

_[Original pull request](https://github.ugent.be/unipept/unipept/pull/649) by @ninewise on Wed Aug 22 2018 at 11:44._
_Merged by @bmesuere on Tue Jan 15 2019 at 14:04._

### CODE REVIEW

**Comment by bmesuere** [comment]
20*16GB RDIMM, 1866 MHz, Standard Volt, Dual Rank, x4 Data Width

_[Original comment](https://github.ugent.be/unipept/unipept/pull/649#issuecomment-78824) by @bmesuere on Wed Aug 22 2018 at 11:56._

File: app/views/api/clidocs/casestudy_metagenomics.html.erb [hunk 1]
```diff
@@ -422,6 +422,35 @@
...
+<!--          -->  umgap uniq |                             # joining equal headers (all taxa of the same read)
+<!--          -->  umgap taxa2agg taxons.tsv |              # aggregating all taxa
+<!--          -->  grep -v '^>' |                           # dropping headers
+<!--          -->  umgap report taxons.tsv                  # make frequency table
+<!--   -->1,332163,Candidatus Solibacter usitatus
+<!--   -->1,392734,Terriglobus roseus
+<!--   -->1,940615,Granulicella tundricola
+        </pre>
+        <p>
+          This command processes the complete Metabenchmark A1 dataset on our computational server
```
Comment (bmesuere) [inline app/views/api/clidocs/casestudy_metagenomics.html.erb]: ... A1 dataset containing xxx paired-end reads...

_[Original comment](https://github.ugent.be/unipept/unipept/pull/649#discussion_r16589) by @bmesuere on Thu Aug 23 2018 at 15:03._

File: app/views/api/clidocs/casestudy_metagenomics.html.erb [hunk 2]
```diff
@@ -422,6 +422,35 @@
...
+          (16 Intel Xeon CPU E5-2650 v2 at 2.60GHz, 20*16GB RDIMM x4 Data Width at 1866MHz, with the 9-mer index loaded in RAM)
+          in 4h22, that is 1840 reads per second.
```
Comment (bmesuere) [inline app/views/api/clidocs/casestudy_metagenomics.html.erb]: reads or paired-end reads?

_[Original comment](https://github.ugent.be/unipept/unipept/pull/649#discussion_r16590) by @bmesuere on Thu Aug 23 2018 at 15:03._

File: app/views/api/clidocs/casestudy_metagenomics.html.erb [hunk 3]
```diff
@@ -422,6 +422,35 @@
...
+<!--          -->  umgap taxa2agg taxons.tsv |              # aggregating all taxa
+<!--          -->  grep -v '^>' |                           # dropping headers
+<!--          -->  umgap report taxons.tsv                  # make frequency table
+<!--   -->1,332163,Candidatus Solibacter usitatus
+<!--   -->1,392734,Terriglobus roseus
+<!--   -->1,940615,Granulicella tundricola
+        </pre>
+        <p>
+          This command processes the complete Metabenchmark A1 dataset on our computational server
+          (16 Intel Xeon CPU E5-2650 v2 at 2.60GHz, 20*16GB RDIMM x4 Data Width at 1866MHz, with the 9-mer index loaded in RAM)
```
Comment (bmesuere) [inline app/views/api/clidocs/casestudy_metagenomics.html.erb]: The Adler VM only has 8 cores and 200GB ram of the total amount.

_[Original comment](https://github.ugent.be/unipept/unipept/pull/649#discussion_r16591) by @bmesuere on Thu Aug 23 2018 at 15:04._

File: app/views/api/clidocs/casestudy_metagenomics.html.erb (hunk 2 above)
Comment (ninewise) [inline app/views/api/clidocs/casestudy_metagenomics.html.erb]: It processed 7360 lines, that's 3680 headers, 2 headers per end, so 1840 reads? Or do we call this paired-end reads?

_[Original comment](https://github.ugent.be/unipept/unipept/pull/649#discussion_r16599) by @ninewise on Thu Aug 23 2018 at 15:36._

**Pull request merged**